job-iks.yaml

```diff
@@ -10,7 +10,7 @@ spec:
       containers:
         - name: kube-bench
           image: aquasec/kube-bench:latest
-          command: ["kube-bench", "--version", "1.13", "node"]
+          command: ["kube-bench", "run", "--targets", "node", "--version", "1.13"]
```

Comment:
We should also update the version here to 1.20 🤙

Reply:
Hey - yes, makes sense thank you :) Just updated.